docs/user/get-started.md

```diff
@@ -0,0 +1,339 @@
+    NAME                                       READY   STATUS    RESTARTS   AGE
+    scylla-0                                   1/1     Running   0          13m
+    scylla-1                                   1/1     Running   0          12m
+    scylla-2                                   1/1     Running   0          10m
+    vald-agent-ngt-0                           1/1     Running   0          5m49s
+    vald-agent-ngt-1                           1/1     Running   0          5m49s
+    vald-agent-ngt-2                           1/1     Running   0          5m49s
+    vald-agent-ngt-3                           1/1     Running   0          5m49s
+    vald-agent-ngt-4                           1/1     Runnnig   0          5m49s
+    vald-discoverer-97c88678b-wj6xn            1/1     Running   0          5m49s
+    vald-gateway-5bf95f8d97-2v76g              1/1     Running   0          5m49s
+    vald-gateway-5bf95f8d97-5wtb2              1/1     Running   0          78s
+    vald-gateway-5bf95f8d97-7d6j7              1/1     Running   0          78s
+    vald-gateway-5bf95f8d97-gx45c              1/1     Running   0          5m49s
+    vald-gateway-5bf95f8d97-kx2c5              1/1     Running   0          78s
+    vald-gateway-5bf95f8d97-np2lc              1/1     Running   0          5m49s
+    vald-manager-backup-6c9695b69b-9xngp       1/1     Running   0          5m49s
+    vald-manager-backup-6c9695b69b-jvwft       1/1     Running   0          5m49s
+    vald-manager-backup-6c9695b69b-mjs2r       1/1     Running   0          5m49s
+    vald-manager-compressor-6c95bdbfb5-m5t7t   1/1     Running   0          5m49s
+    vald-manager-compressor-6c95bdbfb5-q8hc6   1/1     Running   0          5m49s
+    vald-manager-compressor-6c95bdbfb5-zp8hb   1/1     Running   0          5m49s
+    vald-manager-index-59676f54bb-nzfwt        1/1     Running   0          5m49s
+    vald-meta-559744db-bcrdw                   1/1     Running   0          5m49s
+    vald-meta-559744db-hz7gd                   1/1     Running   0          5m49s
+    ```
+    </details>
+
+### Run
+
+This chapter shows the procedure to run Vald with fashion-mnist dataset.
+
+1. Port Forward
+
+    ```bash
+    kubectl port-forward deployment/vald-gateway 8081:8081
+    ```
+
+2. Download dataset
+
+    In this tutorial. we use [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) as a dataset for indexing and search query.
+
+    ```bash
+    # move to working directory
+    cd example/client
+    
+    # get fashion-mnist
+    wget http://ann-benchmarks.com/fashion-mnist-784-euclidean.hdf5
+    ```
+
+3. Running example
+
+    Vald provides multiple langurages client library such as golang, Java, Node.js, Python and so on.<br>
+    In this case, we use [vald-client-go](https://github.com/vdaas/vald-client-go) which is written by golang.
+
+    We use [`example/client/main.go`](../../example/client/main.go) to run the example.
+    This will execute 4 steps.
+    1. init
```

Comment:
we can delete this section.

Reply:
User do not need to understand the init package in order to do this tutorial, and also the sentense `We use [`example/client/main.go`](../../example/client/main.go) to run the example.` is enough to describe we are using the vald-client-go.